file_times.go

```diff
@@ -134,20 +134,32 @@ func (times *FileTimes) CheckOne(path string) (err error) {
 
 func (time FileTime) Check() (err error) {
 	stat, err := os.Stat(time.Path)
+	lstat, lerr := os.Lstat(time.Path)
```

Comment:
can you extract the Stat+Lstat function to getLatestModTime(file) function?

Reply:
I could!